backend/remote/remote.go

```diff
@@ -168,29 +169,35 @@ func (rf *Factory) Create(address string) (types.Backend, error) {
 		return nil, err
 	}
 
-	go r.monitorPing(rpc)
+	go r.monitorPing(rpc, dataAddress /*use replica data addr for ping tracking*/)
```

Comment:
technically, the comment is just for one parameter... I can move it if you think it's better